run.py

```diff
@@ -161,20 +206,25 @@ class _CLIArgs:
 def read_cli_arguments(envs: Envs) -> _CLIArgs:
     """コマンドライン引数を読み込む。"""
     parser = argparse.ArgumentParser(description="VOICEVOX のエンジンです。")
-    # Uvicorn でバインドするアドレスを "localhost" にすることで IPv4 (127.0.0.1) と IPv6 ([::1]) の両方でリッスンできます.
-    # これは Uvicorn のドキュメントに記載されていない挙動です; 将来のアップデートにより動作しなくなる可能性があります.
-    # ref: https://github.com/VOICEVOX/voicevox_engine/pull/647#issuecomment-1540204653
     parser.add_argument(
         "--host",
         type=str,
-        default="localhost",
-        help="接続を受け付けるホストアドレスです。",
+        help="接続を受け付けるホストアドレスです。指定しない場合、代わりに環境変数 VV_HOST の値が使われます。",
     )
     parser.add_argument(
-        "--port", type=int, default=50021, help="接続を受け付けるポート番号です。"
+        "--port",
+        type=int,
+        help="接続を受け付けるポート番号です。指定しない場合、代わりに環境変数 VV_PORT の値が使われます。",
     )
     parser.add_argument(
-        "--use_gpu", action="store_true", help="GPUを使って音声合成するようになります。"
+        "--use_gpu",
+        nargs="?",
+        const=True,
+        type=argstr_to_bool,
```

Comment:
argparse.BooleanOptionalActionを使うのどうでしょう？

```suggestion
    parser.add_argument(
        "--use_gpu",
        action=argparse.BooleanOptionalAction,
        default=False,
```

ちなみにヘルプはこういう感じになります：

```bash
  --use_gpu, --no-use_gpu
    GPUを使って音声合成するか設定します。指定しない場合、代わりに環境変数 VV_USE_GPU の値が使われます。VV_USE_GPU の値が1の場合はGPUを使用し、0または空文字、値がない場合は使用されません。
```

調べてみた感じ、Linux的には否定のための引数を設けることが多そうでした。
gitのpullの`--[no-]recurse-submodules`とか。

`=True`とか`=0`とかは何を渡せば良いのかが混乱するので、まあこっちのが良いかなーと思っています。
コードも短くなるのと、将来的にTyperとかに移ったときに挙動を一致させられるのが大きそうです。

`no-use_gpu`という名前が良いかはちょっと微妙ですが。。

Reply:
修正しました。

`--use_gpu=false`のように設定できるようにした理由は`--no-use_gpu`という引数があまり良くないように感じたからです。
あとはどうせDockerでしか使わないのでDockerに合わせた感じですね。(判定に使う文字列もDockerが使用しているgoに大体合わせたはず)。
他のフレームワークの動作に合わせるという観点からみると`--no-*`の方がいいですね。